aptihramy-backend/api.py

```diff
@@ -0,0 +1,88 @@
+from fastapi import FastAPI, Depends, Query, Form
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi import FastAPI, File, UploadFile, HTTPException
+from blitzbeaver.literals import ID
+from fastapi.responses import FileResponse
+from database import Database
+import os
+from database_config import get_database
+from constants import COLUMN_RAW_TO_PRETTY, COLUMN_PRETTY_TO_RAW
+from pydantic import BaseModel
+from typing import Dict, List
+import time as time
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+from typing import Dict
+import ast
+import json
+
+app = FastAPI()
+
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+class FilterResponse(BaseModel):
+    data: Dict[ID, List[List[str]]]
+
+
+class FilterRequest(BaseModel):
+    data: Dict[str, str]
```

Comment:
It can be a good idea to create a `models` file (or even folder) to put all the models of input/output format in a separate place. This file is generally messy enough already.